NCEAS/metacatui
# Serialize shortName element in EML

Fixes issue #2536

### To test:
1. Locally and pointing a test node, set a default shortName in the EML model:
https://github.com/NCEAS/metacatui/blob/dae8989595efc14c84f1338ff40a3f3526580b7e/src/js/models/metadata/eml211/EML211.js#L64
2. Create/edit a dataset that doesn't have a short name, and see that the default short name is added to the EML (also visible in the dataset view under "General"):
<img width="984" alt="image" src="https://github.com/user-attachments/assets/6c544e86-d7dc-42a1-8e37-735a89d2b205" />
3. Reset the default shortname to `null`
4. Edit the dataset again and see that the shortname persists
